presto-memory/src/test/java/com/facebook/presto/plugin/memory/TestMemorySmoke.java

```diff
@@ -154,10 +149,10 @@ private void assertQueryResult(String sql, Object... expected)
         }
     }
 
-    private void assertQuery(String sql, String expected)
+    private void assertQueryEquals(String sql, String expected)
```

Comment:
You can simply delete this method so that it uses `assertQuery()` from the base class